features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,19 @@
+When("I visit the site") do
+    visit root_path
+  end
+
+When("I click on {string}") do |string|
```

Comment:
Why do you have all of these step defs that are not used in the feature file?

Reply:
These extra step defs were brought in from an old project.  I can remove them.

Reply:
You don't click on anything in this feature so please remove the step on line 5

Reply:
Done!